internal/ethapi/transaction_args.go

```diff
@@ -445,10 +445,11 @@ func (args *TransactionArgs) ToMessage(globalGasCap uint64, baseFee *big.Int, ex
 		gas = globalGasCap
 	}
 	var (
-		gasPrice   *big.Int
-		gasFeeCap  *big.Int
-		gasTipCap  *big.Int
-		blobFeeCap *big.Int
+		gasPrice            *big.Int
+		gasFeeCap           *big.Int
+		gasTipCap           *big.Int
+		blobFeeCap          *big.Int
+		maxFeeInFeeCurrency *big.Int
```

Comment:
Adding a `// Celo specific` comment in between will reduce the diff size because it breaks the formatter's alignment into two parts. Not sure if that is overall better in this case.

Reply:
Done in dbb2c5759723